koji/flatten-el8/koji-ansible/roles/grobisplitter/files/rhel8-split.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+HOMEDIR=/root/koji-sync/podman/
+BINDIR=/usr/local/bin
+
+ARCHES="x86_64"
+DATE=$(date -Ih | sed 's/+.*//')
+
+DATEDIR=${HOMEDIR}/koji/${DATE}
+
+if [ -d ${DATEDIR} ]; then
+    echo "Directory already exists. Please remove or fix"
+    exit
+else
+mkdir -p ${DATEDIR}
+fi
+
+for ARCH in ${ARCHES}; do
+    # The archdir is where we daily download updates for rhel8
+    ARCHDIR=${HOMEDIR}/repos/
+    if [ ! -d ${ARCHDIR} ]; then
+	echo "Unable to find ${ARCHDIR}"
+	exit
+    fi
+
+    # We consolidate all of the default repositories and remerge them
+    # in a daily tree. This allows us to point koji at a particular
+    # day if we have specific build concerns.
+    OUTDIR=${DATEDIR}/${ARCH}
+    mkdir -p ${OUTDIR}
+    if [ ! -d ${OUTDIR} ]; then
+	echo "Unable to find ${ARCHDIR}"
+	exit
+    else
+	cd ${OUTDIR}
+    fi
+
+    # Begin splitting the various packages into their subtrees
+    ${BINDIR}/splitter.py --action hardlink --target RHEL-8-001 ${ARCHDIR}/BaseOS/ &> /dev/null
```

Comment:
Since it's not really RHEL, should this use `EL` or `CentOS`?

Reply:
*shrug*  My assumption is, they had access in fedora to the RHEL repo internally to their infra, so they named it this way.  I'd prefer not to deviate from their script more than necessary at this time.